4_advanced_usage/10_node_management.txt

```diff
@@ -25,7 +25,13 @@ a full reinstallation of the Rudder Server.
 
 ====
 
+==== Change the agent run schedule
 
+By default, the agent runs on all nodes every 5 minutes. You can modify this value in
+*Administration* -> *Settings* -> *Agent Run Schedule*, as well as the "splay time"
+across nodes (a random delay that alters scheduled run time, intended to spread
+load across nodes).
 
-
+When reducing notably the run interval length, reporting can be in 'No report' state
+until the next run of the agent, which can take up to the previous (longer) interval.
```

Comment:
I like the text, but could we have it in a special "warning" section to highlight it?